inginious/frontend/plugins/manual_scoring/pages/templates/manual_scoring.html

```diff
@@ -113,19 +113,33 @@ <h4 class="my-1">$:_("Rubric")</h4>
     </table>
         </div>
     <div>
-        <strong>$:_("Current Score: ")<strong style="color: #cc0808" id="output">$data['grade']</strong></strong>
+        <h4><strong>$:_("Current Score: ")<strong id="output" style="color: #002a95">$data['grade']</strong></strong></h4>
     </div>
     <div id="gradeEditSubmitStatus"></div>
 </section>
 <!-- Comments -->
-<section class="py-2">
+<section class="py-1">
     <form method="post">
-        <h4 >$:_("Comments")</h4>
-        <div class="py-1">
-        <label for="textComments"></label>
-        <textarea class="form-control" id="textComments" rows="8">$data['comment']</textarea>
+        <h3 class="py-1">
+            $:_("Comments")<i data-toggle="tooltip" data-placement="right"
+                              title="$:_('Use rst format to write the comment')"
```

Comment:
I think this message is not taken in the translation or is it? please double check translations.